From: Zain Wang <zain.wang@rock-chips.com>
Date: Tue, 17 Nov 2015 12:00:45 +0800
Subject: clk: rockchip: add an id for rk3288 crypto clk
Git-commit: 94d5d6a0fbf33c5fde246b61b8c358cfeeba633f
Patch-mainline: v4.5-rc1
References: fate#322003

Add an id for crypto clk to the binding header, so that it can be called
in other part.

Signed-off-by: Zain Wang <zain.wang@rock-chips.com>
Acked-by: Michael Turquette <mturquette@baylibre.com>
Signed-off-by: Heiko Stuebner <heiko@sntech.de>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 include/dt-bindings/clock/rk3288-cru.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/dt-bindings/clock/rk3288-cru.h b/include/dt-bindings/clock/rk3288-cru.h
index c719aacef14f..9e0a5e9acee9 100644
--- a/include/dt-bindings/clock/rk3288-cru.h
+++ b/include/dt-bindings/clock/rk3288-cru.h
@@ -86,6 +86,7 @@
 #define SCLK_USBPHY480M_SRC	122
 #define SCLK_PVTM_CORE		123
 #define SCLK_PVTM_GPU		124
+#define SCLK_CRYPTO		125
 
 #define SCLK_MAC		151
 #define SCLK_MACREF_OUT		152
-- 
2.11.0

